Assets/ScriptableObjects/Plugin Scripts/FragilePlugin.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu]
+public class FragilePlugin : BlockPlugin
+{
+
+    public float TimeToBreak;
+
+    private bool _isDisplaced;
+    private bool _isFirstSteppedOn;
```

Comment:
are these variables only separated because one has movement associated with it as well? If the block does not control movement anymore, I think they can be combined

Reply:
These variables were separated because of the 2 scenarios that would result in the block breaking. I am unsure if merging these variables would help if that's the case (which is why I did not implement the moveable plugin).